src/__tests__/app.test.ts

```diff
@@ -0,0 +1,25 @@
+import * as request from "supertest";
+import app from "../app";
+
+jest.mock("../services/redisSessionStorage");
+jest.mock("../services/apiClientFactory");
+
+describe("Test the root path", () => {
+  test("It should response 200 if forwarded from an HTTPS connection", () => {
+    process.env.NODE_ENV = "production";
```

Comment:
you probably want to clean things up and reset this to its original value after the test, there may be other tests in the future that may rely on the value of this env variable